### ISSUE TITLE
Unhandled Exception 

### ISSUE THREAD

**Issue by KimberleyR** [DynamoDS/Dynamo#7969]
3.0.0 Dynamo version

Windows 10 Operating system

What did you do? 

Ran script. Was running fine yesterday on another machine.

Members of the building to turn red if equal to 1

What did you see instead?

Error message/ Nothing
<img width="322" alt="capture" src="https://user-images.githubusercontent.com/29240536/27164420-49d9f0a2-51e1-11e7-82bb-4b48f683133d.PNG">

**Comment by jnealb** [comment]
@KimberleyR Are you able to share your script with us?

**Comment by KimberleyR** [comment]
Hi there @jnealb Sorry about that. I forgot that they didn't upload when I attached them as I hadn't put them into zip folders. Here are the two files. 
[KJR.DYNAMO.zip](https://github.com/DynamoDS/Dynamo/files/1078966/KJR.DYNAMO.zip)
[KJR.REVIT HOUSE.zip](https://github.com/DynamoDS/Dynamo/files/1078968/KJR.REVIT.HOUSE.zip)

Thanks in advance for any help

**Comment by jnealb** [comment]
Hi @KimberleyR I note that your rvt model is from Revit 2018, but what version of Dynamo are you using? The "ElementFiletr.ByNameContains" node is not resolved with Dynamo 1.3, which is the current version. Thanks for clarifying.

**Comment by KimberleyR** [comment]
Hi there @jnealb 
I am infact working in Dynamo 1.3... 
Is there anyway I could back date to a later version of Dynamo to get the script to run? Its for a university project which is due in a week.

**Comment by jnealb** [comment]
@KimberleyR You would have to uninstall 1.3 and install the appropriate version of Dynamo. Which is it?

**Comment by KimberleyR** [comment]
@jnealb Okay. That's fine with me :) Are you aware of which version allows for the "Element.Filter.ByNameContains" to work appropriately?

**Comment by jnealb** [comment]
No, but I can find out. When did you develop the graph?

**Comment by mjkkirschner** [comment]
you can download older installers from www.dynamobuilds.com for DynamoRevit at least.

**Comment by KimberleyR** [comment]
Just a few days ago (Wednesday at around 11am New Zealand time), but it was on another computer with Revit 2018 and Dynamo, so it may not have had Dynamo 1.3? As it was working on the computer I thought that I would be able to transfer the data onto a USB and it would still run on my personal computer. I am aware that those computers do have Revit 2017 installed also

**Comment by KimberleyR** [comment]
[Query_NZS3604.zip](https://github.com/DynamoDS/Dynamo/files/1082173/Query_NZS3604.zip)
This was the original file if that helps? I am unsure if there is anyway you can check to see which version that it was developed in

**Comment by jnealb** [comment]
Thanks, you answered the question. The Dynamo you used was the one bundled with Revit 2018, which is a much earlier version than 1.3. We will advise on Monday, USA EST am.

**Comment by KimberleyR** [comment]
Cool, Thank you @jnealb. Look forward to the answer! :)

**Comment by jnealb** [comment]
@KimberleyR After loading the Package "Beaker" the ElementFiletr.ByNameContains node is resolved and returns correct results. Missing from your graph is the final custom node: "Element.SetParameterByName(CaSe iNSeNSiTiVe)" so I can not tell if it working as expected.

**Comment by KimberleyR** [comment]
@jnealb Sounds great! I will update beaker. I am guessing that Dynamo 1.3. will work fine :) 
I will see what is output from the update. Hopefully it is all resolved. I will let you know. Thank you for your help!! :)

**Comment by KimberleyR** [comment]
Hi there @jnealb GREAT NEWS!!! I have deleted the old nodes for ElementFilter.ByNameContains and re-mapped them and re-input a new node for the "Element.SetParameterByName(CaSe iNSeNsSiTiVe)" and it worked!!! Yay! 
I am a little surprised as I used this node only a week ago and it seemed to work fine, but not to worry; all is well and its back up and running!!! Thank you so much for your help! It is really appreciated!
